### DISCUSSION TITLE
custom css for dynamic mode?

### DISCUSSION THREAD

**Discussion by markusd1984** [darkreader/darkreader#7069] [Q&A]
Is that possible? I want to fix text background color to use black text which is easier to read then white but I only see the custom css option in the Static mode which I don't like. I'm using v5.

Or is it possible to use Static and achieve the same effect as Dynamic? 
(Is there a default config I can paste in to match it and add additional css)

**Comment by Gusted** [comment]
You will most likely end up using dynamic theme but add custom CSS via the [stylus extension](https://github.com/openstyles/stylus/) which is great to add CSS for a longer time of period(using the DR dev tools can end up being removed as we do not guarantee it's storage)

**Reply by awelotta** [reply]
I've used custom CSS extensions, but the colors aren't accurate because they get applied before Dark Mode.

**Reply by Gusted** [reply]
> , but the colors aren't accurate because they get applied before Dark Mode.

?? Did you try Stylus? We currently don't modify any of stylus's CSS.

**Reply by awelotta** [reply]
I used to use an extension called "Live editor for CSS, Less & Sass - Magic CSS". I'm trying out Stylus now, and it does indeed get applied after Dark Mode.

However, the purpose of custom CSS for me is to make all the links yellow/orange in order to be visible with dark mode + blue-light filter on my computer. I often temporarily disable dark mode in order to see things, but this doesn't toggle Stylus, so I liked having my own CSS in Dark Reader so it would toggle with it.

**Reply by Gusted** [reply]
> but this doesn't toggle Stylus

You can add a attribute selector `[data-darkreader-scheme="dark"]` to check if dark reader is toggled.



> so I liked having my own CSS in Dark Reader so it would toggle with it.

I've been very keen against this idea, as users would expect more and more out of this feature, and at some point certain users would somehow use dark reader for this future instead of the dark mode feature. Custom CSS is far from the goal of having a dark mode.

**Comment by awelotta** [comment]
> as users would expect more and more

That makes sense.

Uhm, sorry to treat you like tech support but the attribute selector doesn't work.
```html
a[data-darkreader-scheme="dark"] {
    text-decoration: underline;
    color:yellow;
}
```
is what I have. It doesn't do anything. I've looked at the HTML for some webpages and I can clearly see the <html ... data-darkreader-scheme = "dark">​ tags, so I'm at a loss how to fix it.

________________________________

**Reply by Gusted** [reply]
You've to use the selector as the first one(and then a space), then continue with the original selector.
```css
[data-darkreader-scheme="dark"] a {
    text-decoration: underline;
    color:yellow;
}

**Reply by virginviolet** [reply]
This was hard to find! Finally.
Tried to figure out how to do this for like two hours.
(And yeah, I have never learned CSS / web development properly.)

**Comment by awelotta** [comment]
Ah thank you! It works!
________________________________

**Comment by virginviolet** [comment]
Is it possible to detect dark reader color schemes, like Dracula, Tokyo Night, et cetera?

**Comment by virginviolet** [comment]
And is there any way to make a style rule that is only enabled when Dark Reader is disabled? So that Dark Reader always overrides the Stylus rule. That way, my Style should look good with any Dark Reader color scheme.
